config/stack_ncar.yaml

```diff
@@ -0,0 +1,243 @@
+cmake:
```

Comment:
how different is `stack_ncar.yaml` to `stack_noaa.yaml`?
Just curious.

Reply:
Just the build-jpeg YES|NO, correct.